examples/step-55/step-55.cc

```diff
@@ -515,10 +515,7 @@ namespace Step55
     const FEValuesExtractors::Vector     velocities(0);
     const FEValuesExtractors::Scalar     pressure(dim);
 
-    typename DoFHandler<dim>::active_cell_iterator cell =
-                                                     dof_handler.begin_active(),
-                                                   endc = dof_handler.end();
-    for (; cell != endc; ++cell)
+    for (const auto &cell : dof_handler.begin_active())
```

Comment:
This does not compile because we need `dof_handler.active_cell_iterators()`, as indicated here:
https://cdash.kyomu.43-1.org/viewBuildError.php?buildid=9721
Fix incoming.